src/main/java/ellie/Ui.java

```diff
@@ -26,20 +26,23 @@ public Ui() {
 
     /**
      * Prints a welcome message, the Ellie logo, and a horizontal line.
-     * Also provides a prompt to enter commands.
      */
-    public void hello() {
-        System.out.println("Hello! I'm Ellie, your CS2103T chat bot! I help by tracking your tasks!");
-        System.out.println(logoEllie + "\n" + horizontalLine);
-        System.out.println("What can I do for you? Type 'help' to see available commands! \n");
+    public static String hello() {
+        String helloMessage = "";
+        helloMessage += "Hello! I'm Ellie, your CS2103T chat bot! I help by tracking your tasks!\n";
+        helloMessage += LOGO_ELLIE + "\n" + HORIZONTAL_LINE + "\n";
+        helloMessage += "What can I do for you? Type 'help' to see available commands! \n";
+        return helloMessage;
```

Comment:
Since names representing methods must be verbs and written in camelCase, you may consider changing the method name here to `showHelloMessage()` as a method name.